docs/tutorial/part-four/index.md

```diff
@@ -4,103 +4,84 @@ typora-copy-images-to: ./
 
-**NOTE:** Gatsby’s data layer is powered by GraphQL. For an in-depth tutorial on
-GraphQL, we recommend [How to GraphQL](https://www.howtographql.com/).
+**नोट:** Gatsby की डेटा लेयर GraphQL द्वारा संचालित है। 
+GraphQL के बारे में और गहराई से जानने के लिए [How to GraphQL](https://www.howtographql.com/) देखें।
 
-## Data in Gatsby
+## Gatsby में डेटा
 
-A website has four parts: HTML, CSS, JS, and data. The first half of the
-tutorial focused on the first three. Now let’s learn how to use data in Gatsby
-sites.
+एक वेबसाइट के चार भाग हैं: HTML, CSS, JS और डेटा। ट्यूटोरियल के पहले आधे भाग पहले तीन पर केंद्रित थे। अब हम Gatsby साइटों में डेटा का उपयोग करना सीखेंगे।
 
-**What is data?**
+**डेटा क्या है?**
 
-A very computer science-y answer would be: data is things like `"strings"`,
-integers (`42`), objects (`{ pizza: true }`), etc.
+एक बहुत ही कंप्यूटर विज्ञान-नी उत्तर होगा: डेटा जैसे `"strings"`,
+integers (`42`), objects (`{ pizza: true }`) आदि चीजें हैं।
 
-For the purpose of working in Gatsby, however, a more useful answer is
-"everything that lives outside a React component".
+हालांकि, Gatsby में काम करने के उद्देश्य से, एक अधिक उपयोगी उत्तर है "सब कुछ जो एक React कौम्पोनॅन्ट के बाहर रहता है"।
 
-So far, you've been writing text and adding images _directly_ in components.
-Which is an _excellent_ way to build many websites. But, often you want to store
-data _outside_ components and then bring the data _into_ the component as
-needed.
+अब तक, आप कौम्पोनॅन्ट में टेक्स्ट लिख रहे थे और  _डायरेक्टली_ इमेजस ऐड कर रहे थे जो की अनेक वेबसाइट बनाने का एक _एक्सीलेंट _ तरीका है। लेकिन, अक्सर आप डेटा कौम्पोनॅन्ट के  _बाहर_ स्टोर करना चाहते हैं और फिर जरूरत पड़ने पर डेटा कौम्पोनॅन्ट के _अंदर_ लाते है।
 
-If you're building a site with WordPress (so other contributors
-have a nice interface for adding & maintaining content) and Gatsby, the _data_
-for the site (pages and posts) are in WordPress and you _pull_ that data, as
-needed, into your components.
+यदि आप WordPress (ताकि अन्य योगदानकर्ताओं के लिए कॉंटेंट ऐड करने और बनाए रखने के लिए एक अच्छा इंटरफ़ेस हो) और Gatsby के साथ एक साइट का निर्माण कर रहे हैं, साइट के लिए (पेजेज और पोस्ट) _डेटा_  WordPress में हैं और आप जरूरत पड़ने पर डेटा को कौम्पोनॅन्ट मे _पुल_ करते है।
 
-Data can also live in file types like Markdown, CSV, etc. as well as databases
-and APIs of all sorts.
+डेटा, फाइल्स जैसे Markdown, CSV आदि के साथ-साथ डेटाबेस और सभी प्रकार के APIs में भी रह सकती है।
 
-**Gatsby's data layer lets you pull data from these (and any other source)
-directly into your components**—in the shape and form you want.
+**Gatsby की डेटा लेयर आपको इन (और किसी भी अन्य सोर्स) से डेटा पुल करने देती है सीधे अपने कौम्पोनॅन्ट में** - जिस आकार और रूप में आप चाहते हैं।
 
-## Using Unstructured Data vs GraphQL
+## असंरचित डेटा बनाम GraphQL का उपयोग करना
```

Comment:
`असंरचित` => `अंस्ट्रक्चर्ड`

I am not sure about this. Which one looks better?

Reply:
```असंरचित``` is bit difficult to understand. so, let us have ```अंस्ट्रक्चर्ड```